src/main/java/uk/ac/ebi/ampt2d/accession/server/RestExceptionHandler.java

```diff
@@ -34,6 +37,12 @@
         return buildResponseEntity(new ApiError(HttpStatus.UNSUPPORTED_MEDIA_TYPE, error, ex));
     }
 
+    @ExceptionHandler(ConstraintViolationException.class)
+    protected ResponseEntity<Object> handleConstraintViolationException(ConstraintViolationException ex) {
```

Comment:
Too generic, ConstraintViolationException has enough information to compose a more detailed answer. If this error was creating the object I think Spring has an automated version. If it comes from the persistence layer I'm not sure it should be here.